charts/kubernikus-system/charts/node-problem-detector/templates/daemonset.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: node-problem-detector
+spec:
+  updateStrategy:
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: node-problem-detector
+    spec:
+      tolerations:
+      - key: "species"
```

Comment:
This looks like a leftover from the bare-metal deployment. How do I even read this? If the key `species` exists it adds a `NoSchedule` toleration? Confused.

Reply:
Indeed. Removed it. Not sure we even need it in the bare-metal  deploy.

Read like: If any node taint `Exists` that would cause a `NoSchedule` of the pod it is ignored. In other words, the pods tolerate any node taint and get deployed everywhere.

Reply:
Wat? 🤣